yarn-audit-known-issues

```diff
@@ -1 +1 @@
-{"type":"auditAdvisory","data":{"resolution":{"id":1091647,"path":"aws-sdk>xml2js","dev":false,"bundled":false,"optional":false},"advisory":{"findings":[{"version":"0.4.19","paths":["aws-sdk>xml2js"]}],"metadata":null,"vulnerable_versions":"<=0.4.23","module_name":"xml2js","severity":"high","github_advisory_id":"GHSA-776f-qx25-q3cc","cves":["CVE-2023-0842"],"access":"public","patched_versions":"<0.0.0","cvss":{"score":7.3,"vectorString":"CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:L"},"updated":"2023-04-07T21:00:55.000Z","recommendation":"None","cwe":["CWE-1321"],"found_by":null,"deleted":null,"id":1091647,"references":"- https://nvd.nist.gov/vuln/detail/CVE-2023-0842\n- https://fluidattacks.com/advisories/myers/\n- https://github.com/Leonidas-from-XIV/node-xml2js/\n- https://github.com/advisories/GHSA-776f-qx25-q3cc","created":"2023-04-05T21:30:24.000Z","reported_by":null,"title":"xml2js is vulnerable to prototype pollution ","npm_advisory_id":null,"overview":"xml2js version 0.4.23 allows an external attacker to edit or add new properties to an object. This is possible because the application does not properly validate incoming JSON keys, thus allowing the `__proto__` property to be edited.","url":"https://github.com/advisories/GHSA-776f-qx25-q3cc"}}}
```

Comment:
you'll want to merge main in, and probably regenerate this file, as there is a dependabot PR to fix grpc that I just merged in